core/src/banking_stage.rs

```diff
@@ -473,21 +472,28 @@ impl BankingStage {
                 let data_budget = data_budget.clone();
                 let connection_cache = connection_cache.clone();
                 let bank_forks = bank_forks.clone();
```

Comment:
nit: i think there's no need to create these temporary variables; just `.clone()` them inline at `Forwarder::new`: `data_budget` and `connection_cache`, and `bank_forks`.

Reply:
Done: 43743691d856080a37a4daf1a6a9c037d7d7a9a6